src/main/resources/application.properties

```diff
@@ -3,15 +3,11 @@ info.app.version=1.0
 logging.uris.exclude.regex=/health|/info|/ping|/swagger.*|/webjars/.*|/v2/api-docs
 azure.application-insights.web.enable-W3C=true
 courts=Kingston upon Thames,Westminster,Southwark,City of London,Wimbledon
-
-elite2.uri.root=http://localhost:8080
-elite2.api.uri.root=${elite2.uri.root}/api
-
-auth.endpoint.url=http://localhost:9090/auth
+elite2api.endpoint.url=http://localhost:8080
```

Comment:
ideally we shouldn't be using properties files, everything should be in the yaml files.  Properties files were for before the yaml was invented.  Yaml allows better grouping and nesting of properties.  Obviously don't change, but worth bearing in mind